polaris-react/src/components/Tile/Tile.stories.tsx

```diff
@@ -0,0 +1,177 @@
+import React from 'react';
+import type {ComponentMeta} from '@storybook/react';
+import {Tile, Text} from '@shopify/polaris';
+
+export default {
+  component: Tile,
+} as ComponentMeta<typeof Tile>;
+
+const styles = {
+  border: '1px solid grey',
+  padding: '5px',
```

Comment:
Let's use tokens here

```suggestion
  background: 'var(--p-surface)',
  border: 'var(--p-border-base)',
  borderRadius: 'var(--p-border-radius-2)',
  padding: 'var(--p-space-4)',
```